src/Microsoft.Diagnostics.ExtensionCommands/Host/CommandFormatHelpers.cs

```diff
@@ -16,6 +16,15 @@ public static class CommandFormatHelpers
         public const string DebugHeaderExport = "DotNetRuntimeDebugHeader";
         public const string ContractDescriptorExport = "DotNetRuntimeContractDescriptor";
 
+        public static void DisplaySetttingService(this CommandBase command)
+        {
+            ISettingsService settingsService = command.Services.GetService<ISettingsService>() ?? throw new DiagnosticsException("Settings service required");
+            command.Console.WriteLine("Settings:");
+            command.Console.WriteLine($"-> Use CDAC contract reader: {settingsService.UseContractReader}");
```

Comment:
Likely descriptions would be better from the settings service itself. for another time we likely should be able to iterate through settings.

Reply:
I'm going leave that alone for now.